tests/h2-tests/tests/push_promise.rs

```diff
@@ -256,7 +368,6 @@ fn recv_push_promise_skipped_stream_id() {
 #[test]
 fn recv_push_promise_dup_stream_id() {
```

Comment:
Would you be able to implement this test?

The idea is that the server sends a push promise with stream ID 4 or something. This works.

Then the server attempts to send another push promise with ID 4 and this does not work.